### PR TITLE
Add ibis PCB

### PR DESCRIPTION
**Pull request by hineybush**
## Description

New PCB for my ibis keyboard, finally getting it into QMK

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/hineybush/ibis/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#include QMK_KEYBOARD_H
```
Comment (zvecr) [inline keyboards/hineybush/ibis/keymaps/default/keymap.c:1]:
```suggestion
// Copyright 2022 Josh Hinnebusch (@hineybush)
// SPDX-License-Identifier: GPL-2.0-or-later

#include QMK_KEYBOARD_H
```

File: keyboards/hineybush/ibis/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#include QMK_KEYBOARD_H
```
Comment (zvecr) [inline keyboards/hineybush/ibis/keymaps/via/keymap.c:1]:
```suggestion
// Copyright 2022 Josh Hinnebusch (@hineybush)
// SPDX-License-Identifier: GPL-2.0-or-later

#include QMK_KEYBOARD_H
```

File: keyboards/hineybush/ibis/info.json [hunk 3]
```diff
@@ -0,0 +1,780 @@
...
+    },
+    "processor": "atmega32u4",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0xEAA9",
+        "vid": "0x04D8"
+    },
+    "LED_CAPS_LOCK_PIN": "D2",
+    "LED_NUM_LOCK_PIN": "B7",
```
Comment (zvecr) [inline keyboards/hineybush/ibis/info.json:49]:
```suggestion
    "indicators": {
      "num_lock": "B7",
      "caps_lock": "D2"
    },
```

File: keyboards/hineybush/ibis/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    KC_TRNS,  KC_TRNS, KC_TRNS,                KC_MPLY,                        KC_LALT, KC_TRNS,  KC_RGUI,                      KC_MPRV, KC_VOLD, KC_MNXT,      KC_TRNS, KC_TRNS, KC_TRNS
+    ),
+    [2] = LAYOUT_all(
+    KC_TRNS,  KC_TRNS, KC_TRNS,KC_TRNS, RESET ,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS,     KC_TRNS,KC_TRNS,KC_TRNS, KC_TRNS,
+    KC_TRNS,  KC_TRNS, KC_TRNS,KC_TRNS, RESET ,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS,             KC_TRNS,     KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS,
+    KC_TRNS,  KC_TRNS, KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS,                          KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS,
+    KC_TRNS,  KC_TRNS, KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS,            KC_TRNS,     KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS,
+    KC_TRNS,  KC_TRNS, KC_TRNS,                KC_TRNS,                        KC_TRNS, KC_TRNS,  KC_TRNS,                      KC_TRNS,KC_TRNS,KC_TRNS,        KC_TRNS, KC_TRNS, KC_TRNS
+    )
```
Comment (zvecr) [inline keyboards/hineybush/ibis/keymaps/via/keymap.c:31]:
```suggestion
    ),
    [3] = LAYOUT_all(
    KC_TRNS,  KC_TRNS, KC_TRNS,KC_TRNS, RESET ,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS,     KC_TRNS,KC_TRNS,KC_TRNS, KC_TRNS,
    KC_TRNS,  KC_TRNS, KC_TRNS,KC_TRNS, RESET ,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS,             KC_TRNS,     KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS,
    KC_TRNS,  KC_TRNS, KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS,                          KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS,
    KC_TRNS,  KC_TRNS, KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS,            KC_TRNS,     KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS,
    KC_TRNS,  KC_TRNS, KC_TRNS,                KC_TRNS,                        KC_TRNS, KC_TRNS,  KC_TRNS,                      KC_TRNS,KC_TRNS,KC_TRNS,        KC_TRNS, KC_TRNS, KC_TRNS
    )
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/hineybush/ibis/keymaps/default/keymap.c (hunk 1 above)
Comment (hineybush) [reply keyboards/hineybush/ibis/keymaps/default/keymap.c:1]: y'all should add this back in to`qmk new_keyboard`

Comment (zvecr) [reply keyboards/hineybush/ibis/keymaps/default/keymap.c:1]: We accept PRs to the tooling :stuck_out_tongue_winking_eye:

**Review by drashna — APPROVED** [review]

File: keyboards/hineybush/ibis/readme.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# hineybush/ibis
+
+![hineybush/ibis](https://i.imgur.com/TaCryDr.png)
```
Comment (fauxpark) [inline keyboards/hineybush/ibis/readme.md:3]:
```suggestion
![hineybush/ibis](https://i.imgur.com/TaCryDrh.png)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**